ertviz/models/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 
 def _convertdate(dstring):
-    return datetime.strptime(dstring, "%Y-%m-%d %H:%M:%S")
+    return datetime.strptime(dstring, "%a, %d %b %Y %H:%M:%S GMT")
```

Comment:
should we not use ISO8601 for the time representation? 
https://www.w3.org/TR/NOTE-datetime

i.e something like 1997-07-16T19:20:30.45+01:00 and then let the browser ( wherever located ) handle how to represent this in the html?

Reply:
This is the format currently coming from ert-storage, so it would have to be fixed there

Reply:
I'll create an issue on it for art-storage - at least to look at how it is sent. Because this is not the best way to send it right?

Reply:
I think it more sane to send them the timestamps as integers? @dotfloat, any opinions?